torch/linalg/__init__.py

```diff
@@ -80,96 +88,97 @@
 
             [[ 1.3910,  0.0000],
             [-1.8509,  1.1014]]], dtype=torch.float64)
-    >>> torch.allclose(torch.matmul(l, l.transpose(-2, -1)), a)
+    >>> torch.allclose(l @ l.transpose(-2, -1), a)
     True
 """)
 
 inv = _add_docstr(_linalg.linalg_inv, r"""
 linalg.inv(input, *, out=None) -> Tensor
 
-Computes the multiplicative inverse matrix of a square matrix :attr:`input`, or of each square matrix in a
-batched :attr:`input`. The result satisfies the relation:
+Computes the `inverse`_ of square matrix or batch of matrices
```

Comment:
"of square matrix or batch of matrices" -> "of a square matrix or of a batch of square matrices."